scout/server/blueprints/cases/templates/cases/utils.html

```diff
@@ -312,7 +311,7 @@ <h5 class="modal-title" id="exampleModalLabel">Beacon submission</h5>
                               institute_id=variant.institute,
                               case_name=case.display_name,
                               variant_id=variant._id) }}">
-    {{ variant.sub_category|upper }}({{ variant.chromosome }}{{ variant.cytoband_start }}-{{ variant.chromosome }}{{ variant.cytoband_end }})
+    {{ variant.sub_category|upper }}({{ variant.chromosome }}{{ variant.cytoband_start }}-{{ variant.end_chrom }}{{ variant.cytoband_end }})
```

Comment:
```suggestion
    {{ variant.sub_category|upper }}({{ variant.chromosome }}{{ variant.cytoband_start }}::{{ variant.end_chrom }}{{ variant.cytoband_end }})
```

Reply:
:: is really ISCN2016, but the new 2020 nomenclature is silly complex using like chr:pos_posdelins[chr:pos_pos] (see eg https://varnomen.hgvs.org/recommendations/DNA/variant/complex/) Either is fine. 😸

Reply:
Who am I to challenge ISCN2016? 😄